bin/swap-slots

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+RESOURCE_GROUP=$1
+APP_NAME="$RESOURCE_GROUP-as"
+
+az webapp deployment slot swap --name "$APP_NAME" --slot staging --resource-group "$RESOURCE_GROUP"
```

Comment:
Should this also restart the app service, then?

Reply:
Yeah, probably makes sense to do that too